package.json

```diff
@@ -52,7 +52,7 @@
     "pinia": "^2.0.35",
```

Comment:
Note that an old version of the shared Fee Summary is imported above. (The new version must have a breaking changes as it's 2.x.x.) The Fee Summary "depends" on an old version of sbc-common-components -- I don't know why exactly -- but it seems to work fine in the preview site so I'm going to leave this alone for now.